proto/kava/community/v1beta1/params.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+package kava.community.v1beta1;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/timestamp.proto";
+
+option go_package = "github.com/kava-labs/kava/x/community/types";
+
+// Params defines the parameters of the community module.
+message Params {
```

Comment:
should a query be added for getting x/community params?

Reply:
yeah, realised there a few missing pieces